### PR TITLE
Deduplicate more python scripts based on their intention

### PR DESCRIPTION
**Pull request by cpuhrsch**

### CODE REVIEW

**Comment by zou3519** [comment]
WAIT

File: python/ubenchmarks/nn_comp.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
         # numbers = numbers.transpose(0, 1)
         # numbers = numbers.sigmoid()
         # numbers = numbers.transpose(0, 1)
-        # numbers = numbers.tanh()
+        numbers = numbers.tanh()
```
Comment (zou3519) [inline python/ubenchmarks/nn_comp.py]: Did you mean to commit this?

Comment (cpuhrsch) [reply python/ubenchmarks/nn_comp.py]: Yes

**Comment by cpuhrsch** [comment]
@zou3519 - yes, I had briefly added this as a submodule to look at whether it's possible to pull in foreign benchmarks and interface them. it doesn't seem worth the trouble.

**Comment by apaszke** [comment]
Do those files exist somewhere else as well?

**Comment by cpuhrsch** [comment]
@apaszke - Yes, they're part of the toplevel 'rnn' folder whose files be integrated into our benchmark suite once it's setup.

**Pull request merged**